components/style/error_reporting.rs

```diff
@@ -27,10 +28,14 @@ pub trait ParseErrorReporter {
 /// TODO(emilio): The name of this reporter is a lie, and should be renamed!
 pub struct StdoutErrorReporter;
 impl ParseErrorReporter for StdoutErrorReporter {
-    fn report_error(&self, input: &mut Parser, position: SourcePosition, message: &str) {
+    fn report_error(&self, input: &mut Parser, position: SourcePosition, message: &str, servo_url : Option<&ServoUrl>) {
          if log_enabled!(log::LogLevel::Info) {
              let location = input.source_location(position);
-             info!("{}:{} {}", location.line, location.column, message)
+             match  servo_url {
```

Comment:
Nit: double spaces

Also, we probably don't want `match` here. We could simply rewrite the existing log statement with `map` and `unwrap` methods of `Option<T>` like,

``` rust
info!("{}{}:{} {}", servo_url.map(|u| &format!("ServoUrl: {}\n", u.as_str())).unwrap_or(""), /* other things */);
```